src/histolab/tiler.py

```diff
@@ -834,12 +827,12 @@ def _scores(self, slide: Slide) -> List[Tuple[float, CoordinatePair]]:
             List of tuples containing the score and the extraction coordinates for each
             tile. Each tuple represents a tile.
         """
-        if next(self._tiles_generator(slide), None) is None:
+        if next(super()._tiles_generator(slide), None) is None:
```

Comment:
why? not needed actually. if you want to be explicit then ok

Reply:
nope cause if i don't call the super method it falls into a cricular reference call ... 🗡️

Reply:
ok I get it